app/controllers/flexible_criteria_controller.rb

```diff
@@ -4,7 +4,7 @@ class FlexibleCriteriaController < ApplicationController
 
   def download
     @assignment = Assignment.find(params[:assignment_id])
-    criteria = @assignment.get_criteria(:all, :flexible)
+    criteria = @assignment.criteria.where(type: 'FlexibleCriterion')
```

Comment:
Now that this is one association, you can do [`destroy_all`](https://apidock.com/rails/ActiveRecord/Relation/destroy_all).

Reply:
Why do we want to call `destroy_all` in this download function?

Reply:
Whoops sorry, I think I meant to type this comment somewhere else! I'll make a note to check on my next review.